### DISCUSSION TITLE
Version note is note saved if save() function is called in event listeners

### DISCUSSION THREAD

**Discussion by neha04** [pimcore/pimcore#8028] [Q&A / Support]
Hi All,

I am tring to save an object in event listener. I have also added versionnote in save fn like 
$object->save(["versionNote" => "Auto Classification"]);
The object is saved but version note is not saved.

I am using latest pimcore version.

**Comment by dvesh3** [comment]
which event are you using to save the object?

**Reply by neha04** [reply]
workflow events like 'workflow.product_workflow.completed' and post add object event.

**Reply by dvesh3** [reply]
sorry, it works for me on 6.8.7:
![image](https://user-images.githubusercontent.com/5137917/106579006-77f83680-6540-11eb-8651-21433e817680.png)

![image](https://user-images.githubusercontent.com/5137917/106578985-6f9ffb80-6540-11eb-9fbb-560796b8c725.png)

**Reply by neha04** [reply]
But it is not working for me.
```
    public function setTransitionDateAndTime(Event $event) {
        $object = $event->getSubject();
        $object->save(["versionNote" => 'Auto Classification']);
    }
```

```
  public static function getSubscribedEvents() {
        return array(
            'workflow.product_workflow.completed' => 'setTransitionDateAndTime'
}
```
![Screenshot from 2021-02-02 15-31-33](https://user-images.githubusercontent.com/21308144/106584151-ddade800-656b-11eb-9934-0e1cabd1da33.png)


Is there any other configuration that can cause this like symfony version?

**Reply by cmann90** [reply]
Hi Neha,
did you find a solution for this?
I have the same issue with Pimcore 6.9.6
Version is saved but note is missing.
`$product->save(["versionNote" => "Just a test"]);`

Setting a note works with the `saveVersion` method but it does not retain the published status as I need it.
`$product->saveVersion(true, true, $versionNote); `
